torch/ao/quantization/_dbr/quantization_state.py

```diff
@@ -157,7 +157,9 @@ def reset_to_new_call(self):
         """
         Resets the internal op counter to start a new top level module call
         """
-        self.idx = 0
+        # torch.nn.Module __setattr__ has overhead,
+        # this code is the explicit fast path for `self.idx = 0`
+        object.__setattr__(self, 'idx', 0)
```

Comment:
nit: same comment as before, maybe we can have a set of utility functions for these things

Reply:
added a comment on the reasoning, not sure if adding one more function would increase the overhead, hope that's ok!